triaging.rst

```diff
@@ -375,6 +375,13 @@ Various informational flags about the issue. Multiple values are possible.
 | easy          | Fixing the issue should not take longer than a day for     |
 |               | someone new to contributing to Python to solve.            |
 +---------------+------------------------------------------------------------+
+| easy (C)      | Fixing the issue should not take longer than a day for     |
+|               | someone new contributing to Python, focused on C/C++.      |
++---------------+------------------------------------------------------------+
+| security_issue| The issue would fit as, or is related as a security issue. |
++---------------+------------------------------------------------------------+
+| PEP 3121      | The issue fit as, or is related to the PEP 3123 module     |
```

Comment:
The PEP number here is off, it should be 3121. Also, a few words about the subject of the PEP, and [a link to it](https://www.python.org/dev/peps/pep-3121/), would be very helpful.